index.d.ts

```diff
@@ -0,0 +1,399 @@
+declare module 'plaid' {
+  import { CoreOptions } from 'request';
+
+  type Callback<T> = (err: Error, response: T) => void;
+  type Iso8601DateString = string; // YYYY-MM-DD
+
+  interface AccessTokenFn<T> {
+    (accessToken: string): Promise<T>;
+    (accessToken: string, cb: Callback<T>): void;
+  }
+
+  // OPTIONS /////////////////////////////////////////////////////////////////
+
+  interface ClientRequestOptions {
+    timeout: number;
+  }
+
+  interface ItemRequestOptions {
+    account_ids?: Array<string>;
+  }
+
+  interface TransactionsRequestOptions extends ItemRequestOptions {
+    count?: number;
+    offset?: number;
+  }
+
+  // DATA TYPES //////////////////////////////////////////////////////////////
+
+  interface Account {
+    account_id: string;
+    balances: {
+      available: number | null;
+      current: number | null;
+      limit: number | null;
+    };
+    mask: string | null;
+    name: string | null;
+    official_name: string | null;
+    subtype: string | null;
+    type: string  | null;
+  }
+
+  interface Category {
+    type: string;
+    hierarchy: Array<string>;
+    id: string;
+  }
+
+  interface PlaidError {
+    error_type: string;
+    error_code: string;
+    error_message: string;
+    display_message: string | null;
+  }
+
+  interface Item {
+    available_products: Array<string>;
+    billed_products: Array<string>;
+    error: PlaidError | null;
+    institution_id: string;
+    item_id: string;
+    webhook: string;
+  }
+
+  interface Credential {
+    name: string;
+    label: string;
+    type: 'text' | 'password';
+  }
+
+  interface Institution {
+    credentials: Array<Credential>;
+    has_mfa: boolean;
+    institution_id: string;
+    mfa: Array<string>;
+    name: string;
+    products: Array<string>;
+  }
+
+  interface InstitutionWithDisplayData extends Institution {
+    colors: Map<string, string>;
+    logo: string | null;
+    name_break: number | null;
+    url_account_locked: string | null;
+    url_account_setup: string | null;
+    url_forgotten_password: string | null;
+    video: null; // This field has been deprecated.
```

Comment:
I am fairly certain we no longer return this in api-v2. @pbernasconi can you confirm?

Reply:
👍 this is legacy and is not returned anymore

Reply:
⚡️